mmaction/datasets/activitynet_dataset.py

```diff
@@ -166,6 +175,48 @@ def dump_results(self, results, out, output_format, version='VERSION 1.3'):
                 'external_data': {}
             }
             mmcv.dump(output_dict, out)
+
+            if classifier_input:
+                with open(out) as f_out:
+                    proposal_results = json.load(f_out)['results']
+                videos = proposal_results.keys()
+                video_idx = 0
+
+                out_classifier_input = open(
```

Comment:
This variable is very confusing. Is it out or in? I guess what you mean is this result serves as the input of the following classifier such as SSN. But users will find it confusing since we do not call SSN or PGCN classifier. Maybe call this var and this PR as 'unified_proposals'/'generate unified proposal file'?